api/db/state.js

```diff
@@ -2,6 +2,17 @@ module.exports = () => ({
   state: {
     tableName: 'states',
 
+    format(attributes) {
+      const out = { ...attributes };
+
+      // Stringify.  If undefined, set that instead.  These
+      // must be set to something or else bookshelf will throw
+      out.medicaid_office = JSON.stringify(out.medicaid_office);
+      out.state_pocs = JSON.stringify(out.state_pocs);
+
+      return out;
+    },
```

Comment:
This stringification was being done in the state route handler, but that's not very reusable.  Since we now update a state in the APD route handler too, I figured this should be more centralized.